config/config.example.toml

```diff
@@ -623,4 +623,7 @@ refunds = "hyperswitch-refund-events"
 disputes = "hyperswitch-dispute-events"
 
 [saved_payment_methods]
-sdk_eligible_payment_methods = "card"
\ No newline at end of file
+sdk_eligible_payment_methods = "card"
+
+[separate_three_ds_version_call_enabled_connectors]
```

Comment:
This need not be an env, as it can't be changed in runtime. We can have this check in `impl Connector`